docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[alert-schema]]
+
+== Alert schema
+
+{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices.  In 7.x versions, the index pattern was `.siem-signals-<spacename>-*`.  In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>-*`, and the field names have changed. The following table includes the new names and provides a cross-reference to the previous field names.
+
```

Comment:
We need to add a Beta Disclaimer here, applying to the non-ECS fields.  Something like "the non-ECS fields listed here are [beta] and subject to change."